controllers/metal3.io/action_result.go

```diff
@@ -36,6 +36,16 @@ func (r actionContinueNoWrite) Dirty() bool {
 	return false
 }
 
+type actionDelayed struct {
+	actionContinue
```

Comment:
Just a note that after https://github.com/metal3-io/baremetal-operator/pull/761/commits/5663434797fc88bd5a1c2c524baf8400e7ea5589 this will need to be `actionUpdate` (or possibly we will add a `Dirty()` method + flag).
Whichever PR merges first, we'll need to remember to update the other.

Reply:
Thanks, done